### PR TITLE
constants moved to Pacman

### PR DESCRIPTION
**Pull request by Christian-B**
Avoid duplication of the constants moved in https://github.com/SpiNNakerManchester/PACMAN/pull/297   

Other classes can still import SARK_PER_MALLOC_SDRAM_USAGE from here as it is still there as it is needed for other constants anyway

Can be merged after but not before https://github.com/SpiNNakerManchester/PACMAN/pull/297

### CODE REVIEW

File: spinn_front_end_common/utility_models/multi_cast_command.py [hunk 1]
```diff
@@ -1,127 +1,127 @@
...
-        """
-        return self._payload is not None
-
-    def __repr__(self):
-        return \
-            "MultiCastCommand(time={}, key={}, payload={},"\
-            " time_between_repeat={}, repeats={})".format(
-                self._time, self._key, self._payload,
-                self._delay_between_repeats, self._repeat)
+# Copyright (c) 2017-2019 The University of Manchester
```
Comment (dkfellows) [inline spinn_front_end_common/utility_models/multi_cast_command.py:1]: Why are there any changes to this file at all? (Has something decided to change line endings on us?)

File: unittests/interface/interface_functions/test_output/core1.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (dkfellows) [inline unittests/interface/interface_functions/test_output/core1.json:1]: Why do we have `test_output` in here?

**Comment by alan-stokes** [comment]
conflicts and open issues

**Comment by Christian-B** [comment]
old an unmerged so assuming no longer needed

**Pull request closed without merging**